invenio_app_ils/records/views.py

```diff
@@ -26,7 +26,7 @@
 class UserInfoResource(UserInfoView):
     """Retrieve current user's information."""
 
-    def response(self, user):
+    def success_response(self, user):
```

Comment:
This was done in our previous PR. If you [rebase](https://www.daolf.com/posts/git-series-part-2/) from master, it should disappear :)